src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -27,11 +27,10 @@ import (
 	"github.com/pkg/errors"
 
 	// only register the proto type
+	crioption "github.com/containerd/containerd/pkg/runtimeoptions/v1"
 	_ "github.com/containerd/containerd/runtime/linux/runctypes"
 	_ "github.com/containerd/containerd/runtime/v2/runc/options"
-
-	crioption "github.com/containerd/containerd/pkg/runtimeoptions/v1"
-	oldcrioption "github.com/containerd/containerd/pkg/runtimeoptions/v1"
+	oldcrioption "github.com/containerd/cri-containerd/pkg/api/runtimeoptions/v1"
```

Comment:
This is the critical part of this PR, which actually allows us to keep backward compat with what was done with cri-containerd.

The backward compat code was introduced as part fcc93b00749b6b71c3e3d7e7fa028d96c3226d0f and the offending commit it had been set to the same as the "new" crioption (thus, breaking the backwards compat).